data/human/pirate jobs.txt

```diff
@@ -3711,3 +3711,720 @@ mission "Republic Assassination [2]"
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 1000
+		dialog "As soon as you land on <planet>, the gang leader hands you your payment of <payment> before the group slips off into the crowd."
+
+mission "Pirate bulk smuggle to <planet> [1]"
+	name "Smuggle employees gang to <planet>"
+	color selected "illegal job: selected"
+	color unselected "illegal job: unselected"
+	color unavailable "illegal job: unavailable"
+	job
+	repeat
+	description "A large, illegal operation is looking to move a team of <bunks> employees to their facility on <destination>, and they will pay you <payment> to take them there without being scanned or caught by planetary law enforcement."
+	passengers 11 30 .8
+	illegal 100000 `Your ship is boarded and the employees are escorted to the brig. Their employeer won't be happy to hear this...`
+	to offer
+		random < 350
+		and
+			"reputation: Pirate" > -10
+			"combat rating" > 150
+	source
+		government "Pirate"
+	destination
+		distance 6 12
+		government "Pirate"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on abort
+		"reputation: Pirate" -= 2
+		dialog "You turn the employees away at the spaceport, their employer won't be happy to hear this..."
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 2500
+		dialog "You find your contact on <planet>, and after a lengthy head count, they hand you your payment of <payment>."
+
+mission "Smuggle cartel members to <planet>"
+	color selected "illegal job: selected"
+	color unselected "illegal job: unselected"
+	color unavailable "illegal job: unavailable"
+	job
+	repeat
+	description "This group of <bunks> cartel members and their secutiy guards covert, yet comftorable travel to <destination>, and they will pay you <payment> to take them there without being scanned or caught by planetary law enforcement."
+	passengers 12 24 .8
+	illegal 250000 `Your ship is boarded and a firefight breaks out! After the inside of your <model> is trashed, the remaining guards and cartel members are escorted to the brig and you're hit with a hefty fine. This sure won't go over well...`
+	stealth
+	to offer
+		random < 350
+		and
+			"reputation: Pirate" > 0
+			"combat rating" > 150
+	to accept
+		has "outfit (installed): Luxury Accommodations"
+	source
+		government "Pirate"
+	destination
+		distance 6 12
+		government "Pirate"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on abort
+		"reputation: Pirate" -= 2
+		dialog "You turn the annoyed cartel and their disgruntled guards away, no doubt they'll tell their fellow elite about this."
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 8000
+		dialog "One of the cartel guards hands you your payment of <payment>, before they and the rest of the group enter the <planet> spaceport."
```

Comment:
See above with the before... you could also just remove the comma instead of changing the wording, I guess.